config/environments/development.rb

```diff
@@ -32,6 +32,10 @@
   # number of complex assets.
   config.assets.debug = true
 
+  # Asset digests allow you to set far-future HTTP expiration dates on all assets,
+  # yet still be able to expire them through the digest params.
+  config.assets.digest = true
```

Comment:
Do we need the digest in dev mode?

Reply:
It was just what came from the newer version of rails.  It can still be handy IMO depending on browser.